src/root/views/Country/RiskWatch/ImminentEventsGDACS/index.tsx

```diff
@@ -102,6 +104,13 @@ function ImminentEventsGDACS(props: Props) {
     url: `risk://api/v1/gdacs/${eventId}/exposure`,
   });
 
+  const countryBounds = React.useMemo(
+    () => {
+      let bbox = turfBbox(country?.bbox ?? []);
```

Comment:
Can we even pass empty array on turfBbox function?
Why are we passing bbox on turfBbox?

Reply:
We don't get the bounds format that is supported by mapbox, we transform it to correct format using turf